corehq/apps/users/views/__init__.py

```diff
@@ -549,7 +550,7 @@ def page_context(self):
     }
 
 
-@require_can_edit_web_users
+@require_can_edit_and_view_web_users
```

Comment:
Don't view-only users also be able to page through the list of web users?

Reply:
hmm? yes that's what this permission is changing. it's adding the `and_view` part

Reply:
I was confused here, just read through the code again. Now my understanding is that the `require_can_edit_and_view_foo` methods return true if the user can edit or can view, and then it's the view's responsibility to check the request for `is_view_only` and render appropriately. Is that accurate? If it is, the name doesn't seem accurate, `require_can_edit_or_view_foo` would make more sense.